{"additions": 1, "auther_ref": "fixmuonCut_106", "auther_sha": "b7c6cf1138c826c6fcbe387044944e023254a0ae", "author": "boudoul", "body": "as mentioned in this Issue : #26771 (thanks @christopheralanwest , and also @abdoulline , @mazarkin for noticing ) there was a mistake introduced after the 2019-->2021 migration , a file path was incorrect - This PR is fixing this issue.\r\n\r\nPR validation:\r\nIt should not affect any std WF (where the geometry is read via the DB) , but I ran few events calling explicitely the 2021 XML :\r\nrunTheMatrix.py -l 11623 --command '--geometry Extended2021'\r\n\r\n#### if this PR is a backport please specify the original PR:\r\nBackport of the 11_X PR #26856\r\n \r\n", "branch": "CMSSW_10_6_X", "changed_files": 1, "comments": 3, "commits": 1, "created_at": "1558443909", "deletions": 1, "labels": ["bug-fix", "comparison-pending", "geometry-rejected", "orp-pending", "pending-signatures", "tests-pending", "upgrade-pending"], "milestone": "CMSSW_10_6_X", "number": 26857, "release-notes": [], "review_comments": 0, "state": "open", "title": "Fix file path forgotten after 2019-->2021 migration (backport) ", "updated_at": "1558444250", "user": "boudoul"}